src/Splitio/Services/Client/Classes/SplitFactory.cs

```diff
@@ -79,16 +76,11 @@ private void BuildSplitClient()
                     {
```

Comment:
as we now create `RedisConfigurationValidator`, I suggest to move this validations: `if (_options.CacheAdapterConfig != null && _options.CacheAdapterConfig.Type == AdapterType.Redis)` into that class please, to have all validations in one place.